src/design/plone/contenttypes/patches/baseserializer.py

```diff
@@ -43,9 +43,9 @@ def design_italia_serialize_to_json_call(self, version=None, include_items=True)
             )
             taxonomy_voc = taxonomy.makeVocabulary(self.request.get("LANGUAGE"))
 
-            title = taxonomy_voc.inv_data.get(self.context.tipologia_notizia, None)
+            title = taxonomy_voc.inv_data.get(self.context.tipologia_notizia, "")
```

Comment:
fatto così però se c'è un `None` nella taxopnomy si rompe. Prima c'era `if title and title.satrtswith(...` che comunque non lo faceva rompere.

Al limite:

```suggestion
            - title = taxonomy_voc.inv_data.get(self.context.tipologia_notizia, "")
            + title = taxonomy_voc.inv_data.get(self.context.tipologia_notizia, None) || ""

```

oppure lascerei come prima